.kokoro/release/publish_javadoc11.sh

```diff
@@ -37,7 +37,23 @@ export NAME=bigtable-client-parent
 export VERSION=$(grep ${NAME}: versions.txt | cut -d: -f3)
 
 # cloud RAD generation
+# use javadoc:aggregate for the following modules
+cd bigtable-client-core-parent/
 mvn clean javadoc:aggregate -B -q -P docFX
+cd ../bigtable-dataflow-parent/
+mvn clean javadoc:aggregate -B -q -P docFX
+cd ../bigtable-hbase-2.x-parent/
+mvn clean javadoc:aggregate -B -q -P docFX
+
+# use javadoc:javadoc for the following modules for Standalone doc generation as javadoc:aggregate breaks due to compilation issues
```

Comment:
Are there any plans to fix these compilation issues?

If they're planned as todos, could you file a tracking issue and link it here, something like
```
# TODO (https://github.com/googleapis/java-bigtable-hbase/issues/3783):
# use javadoc:javadoc ...
```

Reply:
@dandhlee Thanks for having a look. I have raised https://github.com/googleapis/java-bigtable-hbase/issues/3843 to track the javadoc fix and have added a TODO as well. Please let me know if these changes look good as a workaround?